### PR TITLE
Documentation build scripts

### PR DESCRIPTION
**Pull request by chicco785**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
-  - 8
+language: python
+cache: pip
+python:
+  - 2.7
 install:
-  - npm install -g markdownlint-cli --save-dev
-before_script:
-  - markdownlint .
+  - pip install Pygments setuptools docutils mock pillow alabaster commonmark recommonmark mkdocs
```
Comment (taliaga) [inline .travis.yml:6]: I'd avoid duplication with build.sh. You could make travis use the external script for example.

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
 script:
-  - true  # add other tests here
+# to be uncommented when the initial content committed.
+#  - mkdocs build --clean --site-dir _build/html --theme readthedocs 
+  - true
+
+matrix:
+  include:
+  ## linters for json and JavaScript files
+    - language: node_js
```
Comment (taliaga) [inline .travis.yml:15]: what's the plan exactly with nodejs and what with python?

**Review by taliaga — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (chicco785) [reply .travis.yml:15]: python is to test the build using same code as readthedocs
nodejs for the linters

File: .travis.yml (hunk 1 above)
Comment (chicco785) [reply .travis.yml:6]: for the moment let's keep it separate, build in travis is not exactly the same as local one (e.g. you don't need to be root)

**Review by taliaga — DISMISSED** [review]

**Pull request merged**